web/Web/web/interfaces/CAL/functions.py

```diff
@@ -182,3 +182,50 @@ def get_scal_info(session):
             return content['info']
     else:
         raise CALServerError(resp['status'])
+
+
+def get_stratum_documents(session):
+    """
+    :param session: current session
+    :return: return JSON list of documents_ids to judge
+    """
+    h = httplib2.Http()
```

Comment:
Let's use the `request` library instead. I'm trying to move away from `httplib2` to `request`

Reply:
Fixed